cartography/intel/aws/ecr.py

```diff
@@ -103,7 +104,8 @@ def _load_ecr_repo_img_tx(
         ON CREATE SET ri.firstseen = timestamp()
         SET ri.lastupdated = $aws_update_tag,
             ri.tag = repo_img.imageTag,
-            ri.uri = repo_img.repo_uri + COALESCE(":" + repo_img.imageTag, '')
+            ri.uri = repo_img.repo_uri + COALESCE(":" + repo_img.imageTag, ''),
+            ri.image_size = repo_img.imageSizeInBytes
```

Comment:
[non-block] While you're here, are there other fields we'd like to include from https://docs.aws.amazon.com/AmazonECR/latest/APIReference/API_DescribeImages.html?

Maybe media manifest type, image pushed at, and last recorded pull time?

Reply:
Sure, I can add those extra fields.

Reply:
Done